freqtrade/templates/subtemplates/sell_trend_full.j2

```diff
@@ -1,3 +1,8 @@
-(qtpylib.crossed_above(dataframe['rsi'], self.sell_rsi.value)) &  # Signal: RSI crosses above sell_rsi
-(dataframe['tema'] > dataframe['bb_middleband']) &  # Guard: tema above BB middle
-(dataframe['tema'] < dataframe['tema'].shift(1)) &  # Guard: tema is falling
+dataframe.loc[
+    (
+        (qtpylib.crossed_above(dataframe['rsi'], self.sell_rsi.value)) &  # Signal: RSI crosses above sell_rsi
+        (dataframe['tema'] > dataframe['bb_middleband']) &  # Guard: tema above BB middle
+        (dataframe['tema'] < dataframe['tema'].shift(1)) &  # Guard: tema is falling
+        (dataframe['volume'] > 0)  # Make sure Volume is not 0
+    ),
+'sell'] = 1
```

Comment:
This should use `exit_long` - `sell` and `buy` should be considered deprecated (although we're currently not warning about it, and will probably continue to support it for quite some time).

Same in the other "full/minimal" templates